data/SftpReadDemoData.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<entity-facade-xml type="demo">
+    <moqui.service.message.SystemMessageType
+            systemMessageTypeId="AnalyticsDataSync"
+            description="Upload the CSV file into Analytics Database"
+            consumeServiceName="co.hotwax.bi.SystemMessageServices.consume#AnalyticsDataSync"
```

Comment:
@diveshdut sir, Should we also consider changing the name of this consume service and name of the service job?